util/src/main/java/org/killbill/billing/util/nodes/DefaultKillbillNodesService.java

```diff
@@ -44,20 +47,27 @@
 
     private static final Logger logger = LoggerFactory.getLogger(DefaultKillbillNodesService.class);
 
+    private final static int TERMINATION_TIMEOUT_SEC = 5;
```

Comment:
Could we make these configurable at least through System properties (just in case...)?

Reply:
Not against it, but these termination timeout are harcoded all over our code already, e.g [here](https://github.com/killbill/killbill/blob/master/tenant/src/main/java/org/killbill/billing/tenant/api/TenantCacheInvalidation.java#L65) or [here](https://github.com/killbill/killbill-commons/blob/master/queue/src/main/java/org/killbill/queue/DefaultQueueLifecycle.java#L126), ...